auditwheel/policy/__init__.py

```diff
@@ -23,9 +24,30 @@ def get_arch_name() -> str:
 _ARCH_NAME = get_arch_name()
 
 
+def _validate_pep600_compliance(policies) -> None:
+    symbol_versions: Dict[str, Dict[str, Set[str]]] = {}
+    lib_whitelist: Set[str] = set()
+    for policy in sorted(policies, key=lambda x: x['priority'], reverse=True):
+        if policy['name'] == 'linux':
+            continue
+        if not lib_whitelist.issubset(set(policy['lib_whitelist'])):
+            raise ValueError('Invalid "policy.json" file')
```

Comment:
How about making the string more explicit on what's wrong with the policy check? That could make it easier to diagnose the issue when editing the JSON.

Reply:
I added more information and tests for that. Is that better now ?

Reply:
Great, thanks!